cmd/trace-agent/config/config.go

```diff
@@ -77,7 +77,7 @@ func prepareConfig(path string) (*config.AgentConfig, error) {
 		return cfg, err
 	}
 	cfg.ConfigPath = path
-	if features.Has("remote_rates") {
+	if coreconfig.Datadog.GetBool("remote_configuration.enabled") {
```

Comment:
Don't you need to set these as known? Also, most stuff is suffixed `_config`, why are you suffixing this `_configuration`?

Reply:
This is an existing flag that is shared with the core-agent, I can't change the name unfortunately. Do I still need to `SetKnown` in that context ?

Reply:
In that case we're good, `*SetDefault` calls `SetKnown` under the hood and I see it's been added here: https://github.com/DataDog/datadog-agent/blob/ba01c560a72151903ab6c1274dc49f809e3a0462/pkg/config/config.go#L284